--- gcl_mislib.lsp	2014-09-06 11:45:30.000000000 -0400
+++ gcl_mislib.lsp.tpd	2016-05-07 23:17:11.116042875 -0400
@@ -130,7 +130,8 @@
 (defun complement (fn)
 #'(lambda (&rest args) (not (apply fn args))))
 
-(defun default-system-banner ()
+(defun default-system-banner () "")
+#|
   (let (gpled-modules)
     (dolist (l '(:unexec :bfd :readline :xgcl))
       (when (member l *features*)
@@ -147,8 +148,9 @@
 	    "Modifications of this banner must retain notice of a compatible license"
 	    "Dedicated to the memory of W. Schelter"
 	    "Use (help) to get some basic information on how to use GCL.")))
+|#
 
- (defun lisp-implementation-version nil
+(defun lisp-implementation-version nil
    (format nil "GCL ~a.~a.~a"
 	   si::*gcl-major-version*
 	   si::*gcl-minor-version*
