--- package.d~	Fri Feb 14 19:38:28 2003
+++ package.d	Sat May  3 22:27:00 2003
@@ -220,10 +220,14 @@
 
 	x = find_package(n);
 	if (x == Cnil) {
- 	   	FEpackage_error(n,"No such package");  
- 		return Cnil; 
-/*  		x = make_package(n, ns, ul,isize,esize);  */
-/*  		goto L;  */
+/* this changes the common lisp 2.0 defined behavior of in-package
+   (throw an error if the package does not exist) so that it mirrors
+   the 1.0 defined behavior (create the package if it does not exist).
+   This patch is intended to be temporary. Axiom's common lisp code
+   should be fixed and this patch removed before shipment -- tpd
+*/
+  		x = make_package(n, ns, ul,isize,esize);  
+  		goto L;  
 	}
 	if (isize) rehash_pack(&(x->p.p_internal),
 		&x->p.p_internal_size,isize);
@@ -243,7 +247,7 @@
 	}
 	for (;  !endp(ul);  ul = ul->c.c_cdr)
 		use_package(ul->c.c_car, x);
-/* L:  */
+L:  
 	sLApackageA->s.s_dbind = x;
 	vs_reset;
 	return(x);
