--- configure.in	Sat Jan 15 14:17:17 2005
+++ configure.in.tpd	Tue Jan 17 09:32:15 2006
@@ -538,7 +538,7 @@
 	# results, and the version is kept in special file).
     
 	if test -r /etc/.relid -a "X`uname -n`" = "X`uname -s`" ; then
-	    system=MP-RAS-`${AWK} '{print $3}' /etc/.relid'`
+	    system=MP-RAS-`${AWK} '{print $3}' '/etc/.relid'`
 	fi
 	if test "`uname -s`" = "AIX" ; then
 	    system=AIX-`uname -v`.`uname -r`
