--- gcl_cmpmain.lsp	2017-05-12 16:29:38.000000000 -0400
+++ gcl_cmpmain.lsp.tpd	2017-05-12 17:30:34.245850266 -0400
@@ -486,7 +486,7 @@
     (setq na  (namestring
 	       (make-pathname :name name :type (pathname-type(first args)))))
    #+(or dos winnt)
-      (format nil "~a -I~a ~a ~a -c -w ~a -o ~a"
+      (format nil "~a -I~a ~a ~a -w -std=gnu89 -c -w ~a -o ~a"
 	      *cc*
 	      (concatenate 'string si::*system-directory* "../h")
 	      (if (and (boundp '*c-debug*) *c-debug*) " -g " "")
