--- configure	2014-10-23 17:29:00.000000000 -0400
+++ configure.new	2017-04-12 13:43:30.194072999 -0400
@@ -2482,8 +2482,8 @@
 
 ac_ext=c
 ac_cpp='$CPP $CPPFLAGS'
-ac_compile='$CC -c $CFLAGS $CPPFLAGS conftest.$ac_ext >&5'
-ac_link='$CC -o conftest$ac_exeext $CFLAGS $CPPFLAGS $LDFLAGS conftest.$ac_ext $LIBS >&5'
+ac_compile='$CC -c -w -std=gnu89 $CFLAGS $CPPFLAGS conftest.$ac_ext >&5'
+ac_link='$CC -o conftest$ac_exeext -w -std=gnu89 $CFLAGS $CPPFLAGS $LDFLAGS conftest.$ac_ext $LIBS >&5'
 ac_compiler_gnu=$ac_cv_c_compiler_gnu
 
 
@@ -2914,7 +2914,7 @@
      *86*darwin*)
      	use=386-macosx
 	if test "$build_cpu" = "x86_64" ; then
-	   CFLAGS="-m64 $CFLAGS";
+	   CFLAGS="-m64 -w -std=gnu89 $CFLAGS";
 	   LDFLAGS="-m64 -Wl,-headerpad,72 $LDFLAGS";
 	else
 	   CFLAGS="-m32 $CFLAGS";
