--- makefile	Tue Dec  7 18:46:09 2004
+++ makefile.tpd	Tue Dec  7 18:47:13 2004
@@ -14,7 +14,8 @@
 PORTDIR = $(shell pwd)
 
 LD_LIBS_PRE=$(FIRST_FILE) $(addprefix -u ,$(PATCHED_SYMBOLS))
-LD_LIBS_POST=$(LIBS) $(LIBC) -lgclp $(LAST_FILE)
+# 20031022000 tpd link axiom's C library code.
+LD_LIBS_POST=$(LIBS) $(LIBC) -lgclp ${OBJ}/${SYS}/lib/libspad.a $(LAST_FILE)
 
 ifeq ($(ARRS),)
 ARRS:=ar rs
@@ -28,7 +29,7 @@
 	rm -rf gmp
 	mkdir gmp
 	a="$^" ; \
-	for i in $^ ; do \
+	for i in $$a ; do \
 		cp $$i gmp/$$(echo $$i | sed -e 's,\.\./,,1' -e 's,/,_,g') ; \
 	done
 	touch $@
