--- gcl_cmpflet.lsp	2004-08-05 18:04:27.000000000 -0400
+++ gcl_cmpflet.lsp.tpd	2005-03-18 22:53:05.000000000 -0500
@@ -390,8 +390,10 @@
           (wt-label *exit*))
     (unwind-no-exit 'tail-recursion-mark)
     (wt-nl "goto TTL;")
-    (cmpnote "Tail-recursive call of ~s was replaced by iteration."
-             (fun-name (car fd))))
+; 20031022000 tpd we don't need to know this
+;    (cmpnote "Tail-recursive call of ~s was replaced by iteration."
+;             (fun-name (car fd)))
+    )
    (t (push-args args)
       (wt-nl (c-function-name "L" (fun-cfun (car fd)) (fun-name (car fd))) "(")
       (dotimes** (n (fun-level (car fd))) (wt "base" n ","))
