--- 386-macosx.defs	2014-10-23 17:29:00.000000000 -0400
+++ 386-macosx.defs.new	2017-04-16 20:50:44.000000000 -0400
@@ -1,5 +1,9 @@
 # powerpc-macosx.defs
 
+# 20170416000 tpd link Axiom's code per Camm's instruction
+SYSTEM_OBJS = ${OBJ}/${SYS}/lib/cfuns-c.o ${OBJ}/${SYS}/lib/sockio-c.o
+OFLAG = -O
+
 # Disable Apple's custom C preprocessor which gets confused when
 # preprocessing some of the *.d files in the o/ subdirectory.
 CC = gcc $(CPPFLAGS)
