--- init_gcl.lsp	Fri May 11 01:56:54 2001
+++ init_gcl.lsp.tpd	Wed Dec 11 23:09:55 2002
@@ -72,12 +72,15 @@
   (cond ((si::get-command-arg "-batch")
          (setq si::*top-level-hook* 'bye))
         ((si::get-command-arg "-f"))
-        (t  ;; if ANY header or license information is printed by the
-            ;; program, then the following License and Enhancement notice
-            ;; must be printed (see License).
-	 (format t "GCL (GNU Common Lisp)  ~A~%~a~%~a~%" "DATE"
-		 "Licensed under GNU Library General Public License"
-		 "Contains Enhancements by W. Schelter")))
+        ((si::get-command-arg "-toploop")
+         (progn (load "toploop" :verbose nil) (toploop)))
+        (t nil))
+;;        (t  ;; if ANY header or license information is printed by the
+;;            ;; program, then the following License and Enhancement notice
+;;            ;; must be printed (see License).
+;;	 (format t "GCL (GNU Common Lisp)  ~A~%~a~%~a~%" "DATE"
+;;		 "Licensed under GNU Library General Public License"
+;;		 "Contains Enhancements by W. Schelter")))
      (setq si::*ihs-top* 1)
   (in-package 'system::user) (incf system::*ihs-top* 2)
   (funcall system::*old-top-level*))
