--- configure	Sat Jan 15 14:17:17 2005
+++ configure.tpd	Sun Apr 16 23:12:48 2006
@@ -6187,7 +6187,7 @@
 echo "configure:6188: checking emacs site lisp directory" >&5
 if [ "$EMACS_SITE_LISP" = "unknown" ] ; then
 	if [ "$EMACS" != "" ] ; then
-		EMACS_SITE_LISP=`$EMACS -q -batch --no-site-file -l conftest.el 2>&1 | sed -e /Loading/d | sed -e /load/d `
+		EMACS_SITE_LISP=`$EMACS -q -batch --no-site-file -l conftest.el 2>&1 | grep -v ^Warning: | sed -e /Loading/d | sed -e /load/d `
 	else
 		EMACS_SITE_LISP=""
 	fi
