--- powerpc-macosx.defs	2008-02-06 16:25:44.000000000 -0500
+++ powerpc-macosx.defs.tpd	2008-02-06 16:23:27.000000000 -0500
@@ -8,6 +8,9 @@
 # This is due to the requirements of the two level namespace.
 LIBS := `echo $(LIBS) | sed -e 's/-lncurses/ /'` /sw/lib/libintl.dylib
 
+# 20080206 tpd link Axiom's code into the image
+EXTRAS = ${OBJ}/${SYS}/lib/cfuns-c.o ${OBJ}/${SYS}/lib/sockio-c.o
+
 # Set this for the linker to operate correctly.
 MACOSX_DEPLOYMENT_TARGET = 10.2
 
