--- makefile	2004-02-21 12:17:00.000000000 -0500
+++ makefile.tpd	2004-02-21 14:47:00.000000000 -0500
@@ -14,7 +14,8 @@
 PORTDIR = $(shell pwd)
 
 LD_LIBS_PRE=$(FIRST_FILE) $(addprefix -u ,$(PATCHED_SYMBOLS))
-LD_LIBS_POST=$(LIBS) $(LIBC) -lgclp $(LAST_FILE)
+# 20031022000 tpd link axiom's C library code.
+LD_LIBS_POST=$(LIBS) $(LIBC) -lgclp ${OBJ}/${SYS}/lib/libspad.a $(LAST_FILE)
 
 ifeq ($(ARRS),)
 ARRS:=ar rs
@@ -154,4 +155,4 @@
 		gazonk*.lsp
 
 .INTERMEDIATE: init_ansi_gcl.lsp.tmp init_gcl.lsp.tmp raw_gcl raw_ansi_gcl
-.PRECIOUS: init_gcl.lsp init_ansi_gcl.lsp
\ No newline at end of file
+.PRECIOUS: init_gcl.lsp init_ansi_gcl.lsp
