--- gcl_mislib.lsp	2017-05-12 16:29:38.000000000 -0400
+++ gcl_mislib.lsp.tpd	2017-05-12 17:20:51.746967001 -0400
@@ -122,7 +122,8 @@
 (defun complement (fn)
 #'(lambda (&rest args) (not (apply fn args))))
 
-(defun default-system-banner ()
+(defun default-system-banner () "")
+#|
   (let (gpled-modules)
     (dolist (l '(:unexec :bfd :readline :xgcl))
       (when (member l *features*)
@@ -139,7 +140,7 @@
 	    "Modifications of this banner must retain notice of a compatible license"
 	    "Dedicated to the memory of W. Schelter"
 	    "Use (help) to get some basic information on how to use GCL.")))
-
+|#
  (defun lisp-implementation-version nil
    (format nil "GCL ~a.~a.~a"
 	   *gcl-major-version*
