--- /usr/lib/gcl-2.6.5/cmpnew/gcl_cmpmain.lsp	2004-09-03 
17:48:54.000000000 +0000
+++ cmpnew/gcl_cmpmain.lsp	2005-01-13 14:29:32.000000000 +0000
@@ -547,7 +547,7 @@
     (setq na  (namestring
 	       (make-pathname :name name :type 
(pathname-type(first args)))))
    #+(or dos winnt)
-      (format nil "~a -I~a ~a ~a -c -w ~a -o ~a"
+      (format nil "~a -I~a ~a ~a -c -w ~s -o ~s"
 	      *cc*
 	      (concatenate 'string si::*system-directory* "../h")
 	      (if (and (boundp '*c-debug*) *c-debug*) " -g " "")
