--- gcl_cmpflet.lsp	Sun Jul 24 12:54:29 2005
+++ gcl_cmpflet.lsp.tpd	Sun Jul 24 13:44:18 2005
@@ -390,8 +390,10 @@
           (wt-label *exit*))
     (unwind-no-exit 'tail-recursion-mark)
     (wt-nl "goto TTL;")
-    (cmpnote "Tail-recursive call of ~s was replaced by iteration."
-             (fun-name (car fd))))
+; 20031022000 tpd we don't need to know this
+;    (cmpnote "Tail-recursive call of ~s was replaced by iteration."
+;             (fun-name (car fd)))
+    )
    (t (push-args args)
       (wt-nl (c-function-name "L" (fun-cfun (car fd)) (fun-name (car fd))) "(")
       (dotimes** (n (fun-level (car fd))) (wt "base" n ","))
