--- makefile~	Thu Mar 20 10:22:55 2003
+++ makefile	Sat May  3 22:30:50 2003
@@ -13,7 +13,7 @@
 PORTDIR = .
 
 LD_LIBS_PRE=$(addprefix -u ,$(PATCHED_SYMBOLS))
-LD_LIBS_POST=$(LIBS) $(LIBC) -lgclp
+LD_LIBS_POST=$(LIBS) $(LIBC) -lgclp ${OBJ}/${SYS}/lib/libspad.a
 
 libgclp.a: $(ODIR)/gcllib.a
 	cp $< $@
@@ -130,4 +130,4 @@
 		$(LSPDIR)/auto_new.lsp foo *maxima* init_*.lsp lib*.a gmp* bfd* *.lsp.tmp
 
 .INTERMEDIATE: init_ansi_gcl.lsp.tmp init_gcl.lsp.tmp raw_gcl raw_ansi_gcl
-.PRECIOUS: init_gcl.lsp init_ansi_gcl.lsp
\ No newline at end of file
+.PRECIOUS: init_gcl.lsp init_ansi_gcl.lsp
