--- mingfile.c	Wed Dec 15 01:49:32 2004
+++ mingfile.c.new	Wed Dec 15 01:49:48 2004
@@ -10,7 +10,7 @@
 	object path;
 	check_arg(1);
 	check_type_or_pathname_string_symbol_stream(&vs_base[0]);
-	path = vs_base[0] = truename(coerce_to_pathname(vs_base[0]));
+	path = vs_base[0] = coerce_to_pathname(vs_base[0]);
 	
 	if (vs_base[0]->pn.pn_name==Cnil && vs_base[0]->pn.pn_type==Cnil) {
 		coerce_to_filename(vs_base[0], filename);
