--- gcl_cmpmain.lsp	2017-04-12 14:23:24.092524999 -0400
+++ gcl_cmpmain.lsp.new	2017-04-12 14:41:47.668460998 -0400
@@ -498,7 +498,7 @@
 	      )
 
    #-(or dos winnt)
-   (format nil  "~a -I~a ~a ~a -c ~s -o ~s ~a"
+   (format nil  "~a -I~a ~a ~a -w -std=gnu89 -c ~s -o ~s ~a"
 	   *cc*
 	   (concatenate 'string si::*system-directory* "../h")
 	   (if (and (boundp '*c-debug*) *c-debug*) " -g " "")

 		  
 		 
