--- 386-macosx.defs	2017-05-12 16:29:28.000000000 -0400
+++ 386-macosx.defs.tpd	2017-05-12 16:40:46.283040906 -0400
@@ -1,5 +1,9 @@
 # powerpc-macosx.defs
 
+# 20170416000 tpd link Axiom's code per Camm's instruction
+SYSTEM_OBJS = ${OBJ}/${SYS}/lib/cfuns-c.o ${OBJ}/${SYS}/lib/sockio-c.o
+OFLAG = -O
+
 # Disable Apple's custom C preprocessor which gets confused when
 # preprocessing some of the *.d files in the o/ subdirectory.
 CC = gcc $(CPPFLAGS)
