--- makefile	2006-08-23 14:14:22.000000000 -0400
+++ makefile.tpd	2008-11-21 13:31:06.000000000 -0500
@@ -15,7 +15,8 @@
 PORTDIR = $(shell pwd)
 
 LD_LIBS_PRE=$(FIRST_FILE) $(addprefix -u ,$(PATCHED_SYMBOLS))
-LD_LIBS_POST=$(LIBS) $(LIBC) -lgclp $(LAST_FILE)
+# 20031022000 tpd link axiom's C library code
+LD_LIBS_POST=$(LIBS) $(LIBC) -lgclp ${OBJ}/${SYS}/lib/libspad.a $(LAST_FILE)
 
 ifeq ($(ARRS),)
 ARRS:=ar rs
