--- gcl_cmpcall.lsp	Sun Jul 24 12:54:28 2005
+++ gcl_cmpcall.lsp.tpd	Sun Jul 24 12:58:36 2005
@@ -264,7 +264,9 @@
             (wt-label *exit*))
       (unwind-no-exit 'tail-recursion-mark)
       (wt-nl "goto TTL;")
-      (cmpnote "Tail-recursive call of ~s was replaced by iteration." fname))
+; 20031022000 tpd we don't need to know this
+;      (cmpnote "Tail-recursive call of ~s was replaced by iteration." fname)
+      )
 
      ;;; Open-codable function call.
      ((and (listp args)
