--- gcl_cmpflet.lsp	2004-02-21 15:51:24.000000000 -0500
+++ gcl_cmpflet.lsp.tpd	2004-02-21 15:51:50.000000000 -0500
@@ -390,8 +390,10 @@
           (wt-label *exit*))
     (unwind-no-exit 'tail-recursion-mark)
     (wt-nl "goto TTL;")
-    (cmpnote "Tail-recursive call of ~s was replaced by iteration."
-             (fun-name (car fd))))
+; 20031022000 tpd we don't need to know this
+;    (cmpnote "Tail-recursive call of ~s was replaced by iteration."
+;             (fun-name (car fd)))
+    )
    (t (push-args args)
       (wt-nl "L" (fun-cfun (car fd)) "(")
       (dotimes** (n (fun-level (car fd))) (wt "base" n ","))
