--- gcl_top.lsp	2017-05-12 16:29:38.000000000 -0400
+++ gcl_top.lsp.tpd	2017-05-12 17:15:50.293482333 -0400
@@ -87,9 +87,9 @@
       (when (if (symbolp *top-level-hook*) (fboundp *top-level-hook*) (functionp *top-level-hook*))
 	(funcall *top-level-hook*)))
 
-    (when (boundp '*system-banner*)
-      (format t *system-banner*)
-      (format t "Temporary directory for compiler files:~%~a~%" *tmp-dir*))
+;    (when (boundp '*system-banner*)
+;      (format t *system-banner*)
+;      (format t "Temporary directory for compiler files:~%~a~%" *tmp-dir*))
 
     (loop
       (setq +++ ++ ++ + + -)
