--- makefile	2003-09-24 11:48:46.000000000 -0400
+++ makefile.tpd	2003-10-23 00:07:38.000000000 -0400
@@ -13,7 +13,8 @@
 PORTDIR = $(shell pwd)
 
 LD_LIBS_PRE=$(addprefix -u ,$(PATCHED_SYMBOLS))
-LD_LIBS_POST=$(LIBS) $(LIBC) -lgclp
+# 20031022000 tpd link axiom's C library code.
+LD_LIBS_POST=$(LIBS) $(LIBC) -lgclp ${OBJ}/${SYS}/lib/libspad.a
 
 libgclp.a: $(ODIR)/gcllib.a
 	cp $< $@
@@ -138,4 +139,4 @@
 		gazonk*.lsp
 
 .INTERMEDIATE: init_ansi_gcl.lsp.tmp init_gcl.lsp.tmp raw_gcl raw_ansi_gcl
-.PRECIOUS: init_gcl.lsp init_ansi_gcl.lsp
\ No newline at end of file
+.PRECIOUS: init_gcl.lsp init_ansi_gcl.lsp
