--- init_gcl.lsp.in	Mon Jan  9 12:24:57 2006
+++ init_gcl.lsp.in.tpd	Mon Apr 17 20:40:05 2006
@@ -85,12 +85,12 @@
    (cond ((si::get-command-arg "-batch")
 	  (setq si::*top-level-hook* 'bye))
 	 ((si::get-command-arg "-f"))
-	 (t (format t si::*system-banner*)
+	 (t (format nil si::*system-banner*)
 	    (let* ((c (find-package "COMPILER"))
 		   (tmp (and c (find-symbol "*TMP-DIR*" c))))
 	      (when tmp
 		(setf (symbol-value tmp) (funcall (find-symbol "GET-TEMP-DIR" c)))
-	(format t "Temporary directory for compiler files set to ~a~%" (symbol-value tmp))))))
+	(format nil "Temporary directory for compiler files set to ~a~%" (symbol-value tmp))))))
    (setq si::*ihs-top* 1)
    (in-package 'system::user) (incf system::*ihs-top* 2)
    (funcall system::*old-top-level*))
