--- mingw.defs	Wed Dec 15 01:46:02 2004
+++ mingw.defs.new	Wed Dec 15 01:46:30 2004
@@ -20,7 +20,7 @@
 FIRST_FILE   = $(ODIR)/firstfile.o
 LAST_FILE    = $(ODIR)/lastfile.o
 LIBFILES     = bcmp.o bzero.o bcopy.o
-EXTRAS       = ${ODIR}/mingwin.o ${ODIR}/mingfile.o
+EXTRAS       = ${ODIR}/mingwin.o ${ODIR}/mingfile.o ${OBJ}/${SYS}/lib/cfuns-c.o ${OBJ}/${SYS}/lib/sockio-c.o
 
 # When using SFASL it is good to have (si::build-symbol-table)
 INITFORM     = (si::build-symbol-table)
