--- alloc.c	Wed Dec 15 01:37:29 2004
+++ alloc.c.new	Wed Dec 15 01:39:06 2004
@@ -1485,7 +1485,17 @@
 	     	RECREATE_HEAP
 #endif
 		;
-	   }
+#ifdef       __MINGW32__
+                /* If malloc() gets called by the C runtime before
+                 * main starts and the shared memory is not yet
+                 * initialised causing boofo.
+                 * SET_REAL_MAXPAGE calls init_shared_memory().
+                 * This problem arose with gcc 3.4.2 and new libs.
+                 */
+                SET_REAL_MAXPAGE
+                ;
+#endif
+            }
 	}
 
 #endif	
