--- cmpaux.c~	Thu Feb 27 09:12:13 2003
+++ cmpaux.c	Sat May  3 22:18:06 2003
@@ -199,28 +199,28 @@
 	return(i);
 }
 
-/* static float */
-/* object_to_float(object x) */
-/* { */
-/* 	float f=0.0; */
+float 
+object_to_float(object x) 
+{ 
+ 	float f=0.0; 
 
-/* 	switch (type_of(x)) { */
-/* 	case t_character: */
-/* 		f = char_code(x);  break; */
-/* 	case t_fixnum: */
-/* 		f = fix(x);  break; */
-/* 	case t_bignum: */
-/* 	case t_ratio: */
-/* 		f = number_to_double(x);  break; */
-/* 	case t_shortfloat: */
-/* 		f = sf(x);  break; */
-/* 	case t_longfloat: */
-/* 		f = lf(x);  break; */
-/* 	default: */
-/* 		FEerror("~S cannot be coerce to a C float.", 1, x); */
-/* 	} */
-/* 	return(f); */
-/* } */
+ 	switch (type_of(x)) { 
+ 	case t_character: 
+ 		f = char_code(x);  break; 
+ 	case t_fixnum: 
+ 		f = fix(x);  break; 
+	case t_bignum: 
+ 	case t_ratio: 
+ 		f = number_to_double(x);  break; 
+ 	case t_shortfloat: 
+ 		f = sf(x);  break; 
+ 	case t_longfloat: 
+ 		f = lf(x);  break; 
+ 	default: 
+ 		FEerror("~S cannot be coerce to a C float.", 1, x); 
+ 	} 
+ 	return(f); 
+ } 
 
 /* static double */
 /* object_to_double(object x) */
