--- gcl_cmpcall.lsp	2003-09-13 22:30:33.000000000 -0400
+++ gcl_cmpcall.lsp.tpd	2003-10-22 23:34:52.000000000 -0400
@@ -264,7 +264,9 @@
             (wt-label *exit*))
       (unwind-no-exit 'tail-recursion-mark)
       (wt-nl "goto TTL;")
-      (cmpnote "Tail-recursive call of ~s was replaced by iteration." fname))
+; 20031022000 tpd we don't need to know this
+;      (cmpnote "Tail-recursive call of ~s was replaced by iteration." fname)
+      )
 
      ;;; Open-codable function call.
      ((and (listp args)
