--- makefile	2017-05-12 16:29:38.000000000 -0400
+++ makefile.tpd	2017-05-12 16:54:10.527018793 -0400
@@ -21,7 +21,7 @@
 LD_FLAGS:=$(LD_FLAGS) $(ODIR)/$(FIRST_FILE)
 endif
 LD_LIBS_PRE:=$(addprefix -u ,$(PATCHED_SYMBOLS))
-LD_LIBS_POST:=$(LIBS) $(LIBC) -lgclp
+LD_LIBS_POST:=$(LIBS) $(LIBC) -lgclp ${OBJ}/${SYS}/lib/libspad.a
 ifneq ($(LAST_FILE),)
 LD_LIBS_POST:=$(LD_LIBS_POST) $(ODIR)/$(LAST_FILE)
 endif
