--- cmpcall.lsp	2003-02-26 17:21:35.000000000 -0500
+++ cmpcall.lsp.tpd	2003-07-13 09:46:46.000000000 -0400
@@ -264,7 +264,9 @@
             (wt-label *exit*))
       (unwind-no-exit 'tail-recursion-mark)
       (wt-nl "goto TTL;")
-      (cmpnote "Tail-recursive call of ~s was replaced by iteration." fname))
+; 20030713000 tpd we don't need to know this
+;      (cmpnote "Tail-recursive call of ~s was replaced by iteration." fname)
+      )
 
      ;;; Open-codable function call.
      ((and (listp args)
