--- init_gcl.lsp.in	2004-02-21 12:17:00.000000000 -0500
+++ init_gcl.lsp.in.tpd	2004-02-21 15:02:45.000000000 -0500
@@ -89,8 +89,8 @@
 	 (bye (if (or compiler::*error-p* (equal result '(nil))) 1 0))))
    (cond ((si::get-command-arg "-batch")
 	  (setq si::*top-level-hook* 'bye))
-	 ((si::get-command-arg "-f"))
-	 (t (format t si::*system-banner*)))
+	 ((si::get-command-arg "-f")))
+;	 (t (format t si::*system-banner*)))
    (setq si::*ihs-top* 1)
    (in-package 'system::user) (incf system::*ihs-top* 2)
    (funcall system::*old-top-level*))
