--- init_gcl.lsp.in	Sun Nov 10 22:43:06 2002
+++ init_gcl.lsp.in.tpd	Sat Mar  8 15:37:52 2003
@@ -81,12 +81,13 @@
   (cond ((si::get-command-arg "-batch")
          (setq si::*top-level-hook* 'bye))
         ((si::get-command-arg "-f"))
-        (t  ;; if ANY header or license information is printed by the
-            ;; program, then the following License and Enhancement notice
-            ;; must be printed (see License).
-	 (format t "GCL (GNU Common Lisp)  ~A~%~a~%~a~%" "DATE"
-		 "Licensed under GNU Library General Public License"
-		 "Contains Enhancements by W. Schelter")))
+        (t nil))
+;;        (t  ;; if ANY header or license information is printed by the
+;;            ;; program, then the following License and Enhancement notice
+;;            ;; must be printed (see License).
+;;	 (format t "GCL (GNU Common Lisp)  ~A~%~a~%~a~%" "DATE"
+;;		 "Licensed under GNU Library General Public License"
+;;		 "Contains Enhancements by W. Schelter")))
      (setq si::*ihs-top* 1)
   (in-package 'system::user) (incf system::*ihs-top* 2)
   (funcall system::*old-top-level*))
