--- configure	2017-05-12 16:29:38.000000000 -0400
+++ configure.tpd	2017-05-12 17:27:40.864997006 -0400
@@ -2494,8 +2494,8 @@
 
 ac_ext=c
 ac_cpp='$CPP $CPPFLAGS'
-ac_compile='$CC -c $CFLAGS $CPPFLAGS conftest.$ac_ext >&5'
-ac_link='$CC -o conftest$ac_exeext $CFLAGS $CPPFLAGS $LDFLAGS conftest.$ac_ext $LIBS >&5'
+ac_compile='$CC -c -w -std=gnu89 $CFLAGS $CPPFLAGS conftest.$ac_ext >&5'
+ac_link='$CC -o conftest$ac_exeext -w -std=gnu89 $CFLAGS $CPPFLAGS $LDFLAGS conftest.$ac_ext $LIBS >&5'
 ac_compiler_gnu=$ac_cv_c_compiler_gnu
 
 
@@ -2927,7 +2927,7 @@
      	use=386-macosx
 	if test "$build_cpu" = "x86_64" ; then
 	   CFLAGS="-m64 $CFLAGS";
-	   LDFLAGS="-m64 -Wl,-headerpad,72 -Wl,-no_pie $LDFLAGS";
+	   LDFLAGS="-m64 -Wl,-headerpad,72 -Wl,-no_pie -w -std=gnu89 $LDFLAGS";
 	else
 	   CFLAGS="-m32 $CFLAGS";
 	   LDFLAGS="-m32  -Wl,-headerpad,56 -Wl,-no_pie $LDFLAGS";
