--- gcl_top.lsp	2013-11-11 09:55:48.000000000 -0500
+++ gcl_top.lsp.tpd	2014-07-02 07:54:30.780280567 -0400
@@ -97,9 +97,10 @@
 	 ((probe-file "init.lsp") (load "init.lsp"))))
       (and (functionp *top-level-hook*)(funcall   *top-level-hook*)))
 
-    (when (boundp '*system-banner*)
-      (format t *system-banner*)
-      (format t "Temporary directory for compiler files set to ~a~%" *tmp-dir*))
+    (setq *tmp-dir* (getenv "TMP"))
+;    (when (boundp '*system-banner*)
+;      (format t *system-banner*)
+;      (format t "Temporary directory for compiler files set to ~a~%" *tmp-dir*))
 
     (loop
       (setq +++ ++ ++ + + -)
