--- init_gcl.lsp.in~	Sun Mar  2 10:23:17 2003
+++ init_gcl.lsp.in	Sat May  3 22:13:38 2003
@@ -77,13 +77,14 @@
    (cond ((si::get-command-arg "-batch")
 	  (setq si::*top-level-hook* 'bye))
 	 ((si::get-command-arg "-f"))
-	 (t  ;; if ANY header or license information is printed by the
-	  ;; program, then the following License and Enhancement notice
-	  ;; must be printed (see License).
-	  (format t "GCL (GNU Common Lisp)  ~A~%~a~%~a~%~%~a~%" "@LI-VERS@"
-		  "Licensed under GNU Library General Public License"
-		  "Dedicated to the memory of W. Schelter"
-		  "Use (help) to get some basic information on how to use GCL.")))
+;;	 (t  ;; if ANY header or license information is printed by the
+;;	  ;; program, then the following License and Enhancement notice
+;;	  ;; must be printed (see License).
+;;	  (format t "GCL (GNU Common Lisp)  ~A~%~a~%~a~%~%~a~%" "@LI-VERS@"
+;;	  "Licensed under GNU Library General Public License"
+;;	  "Dedicated to the memory of W. Schelter"
+;;	  "Use (help) to get some basic information on how to use GCL."))
+   )
    (setq si::*ihs-top* 1)
    (in-package 'system::user) (incf system::*ihs-top* 2)
    (funcall system::*old-top-level*))
