--- roff.nw	Mon Mar 27 18:52:19 2000
+++ roff.nw.tpd	Sun Apr 16 16:50:27 2006
@@ -80,7 +80,7 @@
 other, and quoting each quote is ugly.  The pragmatic solution is to
 copy the awk program into a temporary file, using a shell here-document.
 <<invoke awk program>>=
-awkfile="/tmp/noweb$$.awk"
+awkfile=$(tempfile -p noweb -s .awk)
 trap 'rm -f $awkfile' 0 1 2 10 14 15
 cat > $awkfile << 'EOF'
 <<awk program>>
@@ -662,12 +662,13 @@
 
 base="`basename $1 | sed '/\./s/\.[^.]*$//'`"
 tagsfile="$base.nwt"
+tmpfile=$(tempfile -p tags)
 (echo ".so $macrodir/tmac.w"
 if [ -r "$tagsfile" ]; then 
-   cp $tagsfile /tmp/tags.$$
+   cp $tagsfile $tmpfile
    $AWK '<<action for [[tags]] line>>
-         <<functions>>' /tmp/tags.$$
-   rm -f /tmp/tags.$$
+         <<functions>>' $tmpfile
+   rm -f $tmpfile
  fi
  cat "$@") |
 ($ROFF $opts 2>$tagsfile)
