--- Makefile	2001-03-28 13:40:26.000000000 -0500
+++ Makefile.tpd	2005-05-18 13:27:36.000000000 -0400
@@ -17,14 +17,16 @@
 # MANEXT is the extension for your commands' man pages (usually 1 or l)
 # MAN7EXT is the extension for the nowebstyle man page (usually 7)
 # TEXINPUTS is the directory for TeX macro files
-# ELISP is the directory for emacs lisp files, or /dev/null not to install
-BIN=/usr/local/noweb
-LIB=/usr/local/noweb/lib
-MAN=/usr/local/noweb/man
+# ELISP is the directory for emacs lisp files, or ${TMP}/null not to install
+# We have to go thru a bit of cheese here to get the $ to show up in the
+# final file. 
+BIN:=${AXIOM}/bin/lib
+LIB:=${AXIOM}/bin/lib
+MAN:=${AXIOM}/bin/man
 MANEXT=1
 MAN7EXT=7
 TEXINPUTS=/usr/local/tex/inputs
-ELISP=/dev/null
+ELISP=${TMP}/null
 
 # change WEAVE if you want a different version of noweave to be installed
 WEAVE=noweave		
@@ -37,13 +39,13 @@
 MAN7DIR=$(MAN)/man$(MAN7EXT)
 CATDIR=$(MAN)/cat$(MANEXT)
 CAT7DIR=$(MAN)/cat$(MAN7EXT)
-LIBNAME=$(LIB)
+LIBNAME:=${AXIOM}/bin/lib
 TEXNAME=$(TEXINPUTS)
 
 all: 
-	cd c; make "CC=$(CC)" "CFLAGS=$(CFLAGS)" all 
-	for i in shell lib xdoc tex; do (cd $$i; make all); done
-	cd $(LIBSRC); make "ICONT=$(ICONT)" "ICONC=$(ICONC)" all
+	cd c; ${MAKE} "CC=$(CC)" "CFLAGS=$(CFLAGS)" all 
+	for i in shell lib xdoc tex; do (cd $$i; ${MAKE} all); done
+	cd $(LIBSRC); ${MAKE} "ICONT=$(ICONT)" "ICONC=$(ICONC)" all
 
 FAQ: FAQ.html
 	sleep 1; html2ascii FAQ.html > FAQ
@@ -55,121 +57,127 @@
 
 install: install-code install-man install-tex install-elisp
 
+# There is an unreasonable line here. We want to put the string
+# containing the AXIOM variable into the file without expanding
+# the variable. So we put it into the file with an embedded space
+# to prevent make from recognizing and replacing it. Then we
+# remove the embedded space. I'm sure there must be an easier
+# way but I can't find it.
 install-shell:
-	-mkdir $(BIN) $(LIB) 2>/dev/null
-	sed "s@|LIBDIR|@$(LIBNAME)@" shell/noweb > $(BIN)/noweb
+	-mkdir $(BIN) $(LIB) 2>${TMP}/null
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" shell/noweb > $(BIN)/noweb
 	chmod +x $(BIN)/noweb
-	sed "s@|LIBDIR|@$(LIBNAME)@" shell/notangle > $(BIN)/notangle
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" shell/notangle > $(BIN)/notangle
 	chmod +x $(BIN)/notangle
-	sed "s@|LIBDIR|@$(LIBNAME)@" shell/$(WEAVE) > $(BIN)/$(WEAVE)
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" shell/$(WEAVE) > $(BIN)/$(WEAVE)
 	chmod +x $(BIN)/$(WEAVE)
-	sed "s@|LIBDIR|@$(LIBNAME)@" shell/nountangle > $(BIN)/nountangle
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" shell/nountangle > $(BIN)/nountangle
 	chmod +x $(BIN)/nountangle
-	sed "s@|LIBDIR|@$(LIBNAME)@" shell/nodefs > $(BIN)/nodefs
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" shell/nodefs > $(BIN)/nodefs
 	chmod +x $(BIN)/nodefs
-	sed "s@|LIBDIR|@$(LIBNAME)@" shell/noroots > $(BIN)/noroots
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" shell/noroots > $(BIN)/noroots
 	chmod +x $(BIN)/noroots
-	sed "s@|LIBDIR|@$(LIBNAME)@" shell/nuweb2noweb > $(BIN)/nuweb2noweb
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" shell/nuweb2noweb > $(BIN)/nuweb2noweb
 	chmod +x $(BIN)/nuweb2noweb
-	sed "s@|LIBDIR|@$(LIBNAME)@" shell/cpif > $(BIN)/cpif
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" shell/cpif > $(BIN)/cpif
 	chmod +x $(BIN)/cpif
-	sed "s@|LIBDIR|@$(LIBNAME)@" shell/htmltoc > $(BIN)/htmltoc
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" shell/htmltoc > $(BIN)/htmltoc
 	chmod +x $(BIN)/htmltoc
-	sed "s@|LIBDIR|@$(LIBNAME)@" shell/noroff > $(BIN)/noroff
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" shell/noroff > $(BIN)/noroff
 	chmod +x $(BIN)/noroff
-	sed "s@|LIBDIR|@$(LIBNAME)@" shell/toroff > $(LIB)/toroff
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" shell/toroff > $(LIB)/toroff
 	chmod +x $(LIB)/toroff
 	cp shell/tmac.w $(LIB)
 
 install-code: install-shell
-	-mkdir $(BIN) $(LIB) 2>/dev/null
-	strip c/nt c/markup c/mnt c/finduses
+	-mkdir $(BIN) $(LIB) 2>${TMP}/null
+	#strip c/nt c/markup c/mnt c/finduses
 	cp c/nt c/markup c/mnt c/finduses $(LIB)
-	cd $(LIBSRC); make ICONT=$(ICONT) ICONC=$(ICONC) LIB=$(LIB) BIN=$(BIN) install
-	cd lib; make LIB=$(LIB) install
+	cd $(LIBSRC); ${MAKE} ICONT=$(ICONT) ICONC=$(ICONC) LIB=$(LIB) BIN=$(BIN) install
+	cd lib; ${MAKE} LIB=$(LIB) install
 install-man:
-	-mkdir $(MAN) $(MANDIR) $(MAN7DIR) 2>/dev/null
-	sed -e "s@|LIBDIR|@$(LIBNAME)@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/cpif.1 > $(MANDIR)/cpif.$(MANEXT)
-	sed -e "s@|LIBDIR|@$(LIBNAME)@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/nodefs.1 > $(MANDIR)/nodefs.$(MANEXT)
-	sed -e "s@|LIBDIR|@$(LIBNAME)@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/noroots.1 > $(MANDIR)/noroots.$(MANEXT)
-	sed -e "s@|LIBDIR|@$(LIBNAME)@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/noweb.1 > $(MANDIR)/noweb.$(MANEXT)
-	sed -e "s@|LIBDIR|@$(LIBNAME)@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/noindex.1 > $(MANDIR)/noindex.$(MANEXT)
-	sed -e "s@|LIBDIR|@$(LIBNAME)@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/nuweb2noweb.1 > $(MANDIR)/nuweb2noweb.$(MANEXT)
-	sed -e "s@|LIBDIR|@$(LIBNAME)@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/notangle.1 > $(MANDIR)/notangle.$(MANEXT)
-	sed -e "s@|LIBDIR|@$(LIBNAME)@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/noroff.1 > $(MANDIR)/noroff.$(MANEXT)
-	sed -e "s@|LIBDIR|@$(LIBNAME)@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/sl2h.1 > $(MANDIR)/sl2h.$(MANEXT)
-	sed -e "s@|LIBDIR|@$(LIBNAME)@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/htmltoc.1 > $(MANDIR)/htmltoc.$(MANEXT)
-	sed -e "s@|LIBDIR|@$(LIBNAME)@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/nowebstyle.7 > $(MAN7DIR)/nowebstyle.$(MAN7EXT)
-	sed -e "s@|LIBDIR|@$(LIBNAME)@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/nowebfilters.7 > $(MAN7DIR)/nowebfilters.$(MAN7EXT)
+	-mkdir $(MAN) $(MANDIR) $(MAN7DIR) 2>${TMP}/null
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/cpif.1 > $(MANDIR)/cpif.$(MANEXT)
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/nodefs.1 > $(MANDIR)/nodefs.$(MANEXT)
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/noroots.1 > $(MANDIR)/noroots.$(MANEXT)
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/noweb.1 > $(MANDIR)/noweb.$(MANEXT)
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/noindex.1 > $(MANDIR)/noindex.$(MANEXT)
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/nuweb2noweb.1 > $(MANDIR)/nuweb2noweb.$(MANEXT)
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/notangle.1 > $(MANDIR)/notangle.$(MANEXT)
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/noroff.1 > $(MANDIR)/noroff.$(MANEXT)
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/sl2h.1 > $(MANDIR)/sl2h.$(MANEXT)
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/htmltoc.1 > $(MANDIR)/htmltoc.$(MANEXT)
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/nowebstyle.7 > $(MAN7DIR)/nowebstyle.$(MAN7EXT)
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/nowebfilters.7 > $(MAN7DIR)/nowebfilters.$(MAN7EXT)
 	rm -f $(MANDIR)/noweave.$(MANEXT)
 	(cd $(MANDIR); ln notangle.$(MANEXT) noweave.$(MANEXT))
 	rm -f $(MANDIR)/nountangle.$(MANEXT)
 	(cd $(MANDIR); ln notangle.$(MANEXT) nountangle.$(MANEXT))
 install-gzipped-man:
-	-mkdir $(MAN) $(MANDIR) $(MAN7DIR) 2>/dev/null
-	sed -e "s@|LIBDIR|@$(LIBNAME)@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/cpif.1  | gzip -9 > $(MANDIR)/cpif.$(MANEXT).gz
-	sed -e "s@|LIBDIR|@$(LIBNAME)@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/nodefs.1  | gzip -9 > $(MANDIR)/nodefs.$(MANEXT).gz
-	sed -e "s@|LIBDIR|@$(LIBNAME)@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/noroots.1  | gzip -9 > $(MANDIR)/noroots.$(MANEXT).gz
-	sed -e "s@|LIBDIR|@$(LIBNAME)@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/noweb.1  | gzip -9 > $(MANDIR)/noweb.$(MANEXT).gz
-	sed -e "s@|LIBDIR|@$(LIBNAME)@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/noindex.1  | gzip -9 > $(MANDIR)/noindex.$(MANEXT).gz
-	sed -e "s@|LIBDIR|@$(LIBNAME)@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/nuweb2noweb.1  | gzip -9 > $(MANDIR)/nuweb2noweb.$(MANEXT).gz
-	sed -e "s@|LIBDIR|@$(LIBNAME)@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/notangle.1  | gzip -9 > $(MANDIR)/notangle.$(MANEXT).gz
-	sed -e "s@|LIBDIR|@$(LIBNAME)@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/noroff.1  | gzip -9 > $(MANDIR)/noroff.$(MANEXT).gz
-	sed -e "s@|LIBDIR|@$(LIBNAME)@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/sl2h.1  | gzip -9 > $(MANDIR)/sl2h.$(MANEXT).gz
-	sed -e "s@|LIBDIR|@$(LIBNAME)@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/htmltoc.1  | gzip -9 > $(MANDIR)/htmltoc.$(MANEXT).gz
-	sed -e "s@|LIBDIR|@$(LIBNAME)@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/nowebstyle.7  | gzip -9 > $(MAN7DIR)/nowebstyle.$(MAN7EXT).gz
-	sed -e "s@|LIBDIR|@$(LIBNAME)@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/nowebfilters.7  | gzip -9 > $(MAN7DIR)/nowebfilters.$(MAN7EXT).gz
+	-mkdir $(MAN) $(MANDIR) $(MAN7DIR) 2>${TMP}/null
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/cpif.1  | gzip -9 $(MANDIR)/cpif.$(MANEXT).gz
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/nodefs.1  | gzip -9 $(MANDIR)/nodefs.$(MANEXT).gz
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/noroots.1  | gzip -9 $(MANDIR)/noroots.$(MANEXT).gz
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/noweb.1  | gzip -9 $(MANDIR)/noweb.$(MANEXT).gz
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/noindex.1  | gzip -9 $(MANDIR)/noindex.$(MANEXT).gz
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/nuweb2noweb.1  | gzip -9 $(MANDIR)/nuweb2noweb.$(MANEXT).gz
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/notangle.1  | gzip -9 $(MANDIR)/notangle.$(MANEXT).gz
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/noroff.1  | gzip -9 $(MANDIR)/noroff.$(MANEXT).gz
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/sl2h.1  | gzip -9 $(MANDIR)/sl2h.$(MANEXT).gz
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/htmltoc.1  | gzip -9 $(MANDIR)/htmltoc.$(MANEXT).gz
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/nowebstyle.7  | gzip -9 > $(MAN7DIR)/nowebstyle.$(MAN7EXT).gz
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/nowebfilters.7  | gzip -9 > $(MAN7DIR)/nowebfilters.$(MAN7EXT).gz
 	rm -f $(MANDIR)/noweave.$(MANEXT).gz
 	(cd $(MANDIR); ln notangle.$(MANEXT).gz noweave.$(MANEXT).gz)
 	rm -f $(MANDIR)/nountangle.$(MANEXT).gz
 	(cd $(MANDIR); ln notangle.$(MANEXT).gz nountangle.$(MANEXT).gz)
 install-preformat-man:
 	-echo "Warning: install-preformat-man is obsolete, even on Slackware systems" 1>&2
-	-mkdir $(MAN) $(CATDIR) $(CAT7DIR) 2>/dev/null
-	sed -e "s@|LIBDIR|@$(LIBNAME)@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/cpif.txt  | gzip > $(CATDIR)/cpif.$(MANEXT).gz
-	sed -e "s@|LIBDIR|@$(LIBNAME)@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/nodefs.txt  | gzip > $(CATDIR)/nodefs.$(MANEXT).gz
-	sed -e "s@|LIBDIR|@$(LIBNAME)@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/noroots.txt  | gzip > $(CATDIR)/noroots.$(MANEXT).gz
-	sed -e "s@|LIBDIR|@$(LIBNAME)@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/noweb.txt  | gzip > $(CATDIR)/noweb.$(MANEXT).gz
-	sed -e "s@|LIBDIR|@$(LIBNAME)@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/noindex.txt  | gzip > $(CATDIR)/noindex.$(MANEXT).gz
-	sed -e "s@|LIBDIR|@$(LIBNAME)@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/nuweb2noweb.txt  | gzip > $(CATDIR)/nuweb2noweb.$(MANEXT).gz
-	sed -e "s@|LIBDIR|@$(LIBNAME)@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/notangle.txt  | gzip > $(CATDIR)/notangle.$(MANEXT).gz
-	sed -e "s@|LIBDIR|@$(LIBNAME)@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/noroff.txt  | gzip > $(CATDIR)/noroff.$(MANEXT).gz
-	sed -e "s@|LIBDIR|@$(LIBNAME)@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/sl2h.txt  | gzip > $(CATDIR)/sl2h.$(MANEXT).gz
-	sed -e "s@|LIBDIR|@$(LIBNAME)@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/htmltoc.txt  | gzip > $(CATDIR)/htmltoc.$(MANEXT).gz
-	sed -e "s@|LIBDIR|@$(LIBNAME)@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/nowebstyle.txt  | gzip > $(CAT7DIR)/nowebstyle.$(MAN7EXT).gz
-	sed -e "s@|LIBDIR|@$(LIBNAME)@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/nowebfilters.txt  | gzip > $(CAT7DIR)/nowebfilters.$(MAN7EXT).gz
+	-mkdir $(MAN) $(CATDIR) $(CAT7DIR) 2>${TMP}/null
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/cpif.txt  | gzip > $(CATDIR)/cpif.$(MANEXT).gz
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/nodefs.txt  | gzip > $(CATDIR)/nodefs.$(MANEXT).gz
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/noroots.txt  | gzip > $(CATDIR)/noroots.$(MANEXT).gz
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/noweb.txt  | gzip > $(CATDIR)/noweb.$(MANEXT).gz
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/noindex.txt  | gzip > $(CATDIR)/noindex.$(MANEXT).gz
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/nuweb2noweb.txt  | gzip > $(CATDIR)/nuweb2noweb.$(MANEXT).gz
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/notangle.txt  | gzip > $(CATDIR)/notangle.$(MANEXT).gz
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/noroff.txt  | gzip > $(CATDIR)/noroff.$(MANEXT).gz
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/sl2h.txt  | gzip > $(CATDIR)/sl2h.$(MANEXT).gz
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/htmltoc.txt  | gzip > $(CATDIR)/htmltoc.$(MANEXT).gz
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/nowebstyle.txt  | gzip > $(CAT7DIR)/nowebstyle.$(MAN7EXT).gz
+	sed -e "s@|LIBDIR|@$$ AXIOM/bin/lib@" -e "s@ AXIOM@AXIOM@" -e "s@|TEXINPUTS|@$(TEXNAME)@" xdoc/nowebfilters.txt  | gzip > $(CAT7DIR)/nowebfilters.$(MAN7EXT).gz
 	rm -f $(CATDIR)/noweave.$(MANEXT).gz
 	(cd $(CATDIR); ln notangle.$(MANEXT).gz noweave.$(MANEXT).gz)
 	rm -f $(CATDIR)/nountangle.$(MANEXT).gz
 	(cd $(CATDIR); ln notangle.$(MANEXT).gz nountangle.$(MANEXT).gz)
 install-tex:
-	-mkdir $(TEXINPUTS) 2>/dev/null
+	-mkdir $(TEXINPUTS) 2>${TMP}/null
 	cp tex/nwmac.tex tex/noweb.sty $(TEXINPUTS)
 	-texhash || echo "Program texhash not found or failed"
 
 install-elisp:
-	-mkdir $(ELISP) 2>/dev/null
+	-mkdir $(ELISP) 2>${TMP}/null
 	cp elisp/noweb-mode.el $(ELISP)
 
 checkin:
-	for i in lib tex xdoc; do (cd $$i; make "CINAME=$(CINAME)" "CIMSG=$(CIMSG)" checkin); done
+	for i in lib tex xdoc; do (cd $$i; ${MAKE} "CINAME=$(CINAME)" "CIMSG=$(CIMSG)" checkin); done
 	for i in c icon awk; do (cd $$i; ci -l $(CINAME) $(CIMSG) *.nw Makefile); done
 	(cd elisp; ci -l $(CINAME) $(CIMSG) *.el)
 	ci -l $(CINAME) $(CIMSG) Makefile.nw INSTALL INSTALL.DOS README FAQ COPYRIGHT nwmake *.nw
 
 source: FAQ
-	for i in c shell lib xdoc icon awk tex; do (cd $$i; make CPIF=">" source); done
+	for i in c shell lib xdoc icon awk tex; do (cd $$i; ${MAKE} CPIF=">" source); done
 	sleep 1
-	for i in c shell lib xdoc icon awk tex; do (cd $$i; make touch); done
+	for i in c shell lib xdoc icon awk tex; do (cd $$i; ${MAKE} touch); done
 touch:
 	touch FAQ
-	for i in c shell lib xdoc icon awk tex; do (cd $$i; make touch); done
+	for i in c shell lib xdoc icon awk tex; do (cd $$i; ${MAKE} touch); done
 clean:
-	for i in c shell lib xdoc icon awk tex; do (cd $$i; make clean); done
+	for i in c shell lib xdoc icon awk tex; do (cd $$i; ${MAKE} clean); done
 	rm -f nwsrcfilter *~ */*~
 
 clobber: clean
-	for i in c lib icon awk xdoc; do (cd $$i; make clobber); done
+	for i in c lib icon awk xdoc; do (cd $$i; ${MAKE} clobber); done
 
 Makefile: Makefile.nw
 	chmod +w Makefile
