--- Makefile.nw	Wed Mar 28 13:40:21 2001
+++ noweb.src.Makefile.nw	Wed Nov 27 16:56:48 2002
@@ -22,14 +22,14 @@
 # MANEXT is the extension for your commands' man pages (usually 1 or l)
 # MAN7EXT is the extension for the nowebstyle man page (usually 7)
 # TEXINPUTS is the directory for TeX macro files
-# ELISP is the directory for emacs lisp files, or /dev/null not to install
+# ELISP is the directory for emacs lisp files, or ${TMP}/null not to install
 BIN=/usr/local/noweb
 LIB=/usr/local/noweb/lib
 MAN=/usr/local/noweb/man
 MANEXT=1
 MAN7EXT=7
 TEXINPUTS=/usr/local/tex/inputs
-ELISP=/dev/null
+ELISP=${TMP}/null
 
 # change WEAVE if you want a different version of noweave to be installed
 WEAVE=noweave		
@@ -61,13 +61,13 @@
 install: install-code install-man install-tex install-elisp
 
 install-shell:
-	-mkdir $(BIN) $(LIB) 2>/dev/null
+	-mkdir $(BIN) $(LIB) 2>${TMP}/null
 	<<shell binaries>>
 	cp shell/tmac.w $(LIB)
 
 install-code: install-shell
-	-mkdir $(BIN) $(LIB) 2>/dev/null
-	strip c/nt c/markup c/mnt c/finduses
+	-mkdir $(BIN) $(LIB) 2>${TMP}/null
+	#strip c/nt c/markup c/mnt c/finduses
 	cp c/nt c/markup c/mnt c/finduses $(LIB)
 	cd $(LIBSRC); make ICONT=$(ICONT) ICONC=$(ICONC) LIB=$(LIB) BIN=$(BIN) install
 	cd lib; make LIB=$(LIB) install
@@ -90,19 +90,19 @@
 done
 <<*>>=
 install-man:
-	-mkdir $(MAN) $(MANDIR) $(MAN7DIR) 2>/dev/null
+	-mkdir $(MAN) $(MANDIR) $(MAN7DIR) 2>${TMP}/null
 	<<ordinary pages>>
 @ 
 Slackware no longer uses preformatted compressed pages, just
 compressed pages.
 <<*>>=
 install-gzipped-man:
-	-mkdir $(MAN) $(MANDIR) $(MAN7DIR) 2>/dev/null
+	-mkdir $(MAN) $(MANDIR) $(MAN7DIR) 2>${TMP}/null
 	<<compressed pages>>
 <<*>>=
 install-preformat-man:
 	-echo "Warning: install-preformat-man is obsolete, even on Slackware systems" 1>&2
-	-mkdir $(MAN) $(CATDIR) $(CAT7DIR) 2>/dev/null
+	-mkdir $(MAN) $(CATDIR) $(CAT7DIR) 2>${TMP}/null
 	<<preformatted compressed pages>>
 <<generate chunks>>=
 NORMALPAGES="cpif nodefs noroots noweb noindex nuweb2noweb notangle noroff sl2h htmltoc"
@@ -158,12 +158,12 @@
 done
 <<*>>=
 install-tex:
-	-mkdir $(TEXINPUTS) 2>/dev/null
+	-mkdir $(TEXINPUTS) 2>${TMP}/null
 	cp tex/nwmac.tex tex/noweb.sty $(TEXINPUTS)
 	-texhash || echo "Program texhash not found or failed"
 
 install-elisp:
-	-mkdir $(ELISP) 2>/dev/null
+	-mkdir $(ELISP) 2>${TMP}/null
 	cp elisp/noweb-mode.el $(ELISP)
 
 <<*>>=
