--- totex.nw	Mon Mar 27 18:52:13 2000
+++ totex.nw.tpd	Sun Apr 16 15:46:01 2006
@@ -23,8 +23,11 @@
 nawk '<<awk program for conversion to {\TeX}>>' delay=$delay noindex=$noindex
 @
 On an ugly system, we have to put it in a file.
+
+The awkfile has been changed to use tempfile because creating files
+directly in tmp creates an insecure hole.
 <<invoke awk program using file>>=
-awkfile=/tmp/totex$$.awk
+awkfile=$(tempfile -p totex)
 trap 'rm -f $awkfile; exit 1' 0 1 2 15	# clean up files
 cat > $awkfile << 'EOF'
 <<awk program for conversion to {\TeX}>>
