--- roff.mm	Mon Mar 27 18:52:19 2000
+++ roff.mm.tpd	Sun Apr 16 17:06:44 2006
@@ -214,7 +214,7 @@
 .ADDLIST 1a
 .PRINTLIST
 
-awkfile="/tmp/noweb$$.awk"
+awkfile=$(tempfile -p noweb -s .awk) || { echo "$0: Cannot create temporary file" >&2; exit 1;  }
 trap 'rm -f $awkfile' 0 1 2 10 14 15
 cat > $awkfile \&<< 'EOF'
 \c
@@ -1628,14 +1628,15 @@
 tagsfile="$base.nwt"
 (echo ".so $macrodir/tmac.w"
 if [ -r "$tagsfile" ]; then 
-   cp $tagsfile /tmp/tags.$$
+   tagstemp=$(tempfile -p tags) || { echo "$0: Cannot create temporary file" >&2; exit 1;  }
+   cp $tagsfile $tagstemp
    $AWK '\c
 .USE "action for \*[BEGINCONVQUOTE]tags\*[ENDCONVQUOTE] line" 11c
 \&
          \c
 .USE "functions" 8a
-\&' /tmp/tags.$$
-   rm -f /tmp/tags.$$
+\&' $tagstemp
+   rm -f $tagstemp
  fi
  cat "$@") |
 ($ROFF $opts 2>$tagsfile)
