--- toascii.nw	Mon Mar 27 18:52:17 2000
+++ toascii.nw.tpd	Sun Apr 16 16:59:59 2006
@@ -28,9 +28,9 @@
 Also arranged here is a temporary file for storage of the awk program on an
 ugly system, as discussed below.
 <<arrange temporary files>>=
-awkfile="tmp/awk$$.tmp"
-textfile="/tmp/text$$.tmp"
-tagsfile="/tmp/tags$$.tmp"
+awkfile=$(tempfile -p awk -s .tmp) || { echo "$0: Cannot create temporary file" >&2; exit 1;  }
+textfile=$(tempfile -p text -s .tmp) || { echo "$0: Cannot create temporary file" >&2; exit 1;  }
+tagsfile=$(tempfile -p tags -s .tmp) || { echo "$0: Cannot create temporary file" >&2; exit 1;  }
 export awkfile textfile tagsfile
 trap 'rm -f $awkfile $textfile $tagsfile' 0 1 2 10 14 15
 @ %def textfile tagsfile awkfile
