--- noroff	Wed Mar 28 13:49:22 2001
+++ noroff.tpd	Sun Apr 16 16:53:49 2006
@@ -37,7 +37,7 @@
 tagsfile="$base.nwt"
 (echo ".so $macrodir/tmac.w"
 if [ -r "$tagsfile" ]; then 
-   cp $tagsfile /tmp/tags.$$
+   cp $tagsfile $tmpfile
    $AWK '{
 	     if      (sub(/^###TAG### /       , "")) tags[$1] = $2
 	     else if (sub(/^###BEGINCHUNKS###/, "")) printf ".de CLIST\n.CLISTBEGIN\n"
@@ -88,8 +88,8 @@
 	 #	print str3
 	 #	print convquote(str3)
 	 # }
-	 function tag(s) { if (s in tags) return tags[s]; else return "???" }' /tmp/tags.$$
-   rm -f /tmp/tags.$$
+	 function tag(s) { if (s in tags) return tags[s]; else return "???" }' $tmpfile
+   rm -f $tmpfile
  fi
  cat "$@") |
 ($ROFF $opts 2>$tagsfile)
