--- toascii	Wed Mar 28 13:49:22 2001
+++ toascii.tpd	Sun Apr 16 15:48:52 2006
@@ -7,9 +7,9 @@
                 *)      echo "This can't happen -- $i passed to toascii" 1>&2 ; exit 1 ;;
         esac
 done
-awkfile="tmp/awk$$.tmp"
-textfile="/tmp/text$$.tmp"
-tagsfile="/tmp/tags$$.tmp"
+awkfile=$(tempfile -p awk -s .tmp)
+textfile=$(tempfile -p text -s .tmp)
+tagsfile=$(tempfile -p tags -s .tmp)
 export awkfile textfile tagsfile
 trap 'rm -f $awkfile $textfile $tagsfile' 0 1 2 10 14 15
 nawk 'BEGIN { textfile=ENVIRON["textfile"]
