--- vmlisp.lisp.pamphlet~	Sun Nov 17 14:14:45 2002
+++ vmlisp.lisp.pamphlet	Sun May 18 16:24:37 2003
@@ -8,6 +8,30 @@
 \end{abstract}
 \eject
 \tableofcontents
+\section{The StringImage Fix}
+In GCL 2.5 there is a bug in the write-to-string function.
+It should respect *print-escape* but it does not. That is,
+\begin{verbatim}
+
+In GCL 2.4.1:
+(setq *print-escape* nil)
+(write-to-string '|a|) ==> "a"
+
+In GCL 2.5:
+(setq *print-escape* nil)
+(write-to-string '|a|) ==> "|a|"
+
+\end{verbatim}
+The form2LispString function uses stringimage and fails.
+The princ-to-string function assumes *print-escape* is nil
+and works properly.
+
+<<stringimage fix>>=
+;(define-function 'prin2cvec #'write-to-string)
+(define-function 'prin2cvec #'princ-to-string)
+;(define-function 'stringimage #'write-to-string)
+(define-function 'stringimage #'princ-to-string)
+
 \eject
 <<*>>=
 ; 
@@ -1491,8 +1515,7 @@
 
 ; 24.0 Printing
 
-(define-function 'prin2cvec #'write-to-string)
-(define-function 'stringimage #'write-to-string)
+<<stringimage fix>>
 (define-function 'printexp #'princ)
 (define-function 'prin0  #'prin1)
 
