--- /tmp/axbook/book-contents.xhtml	2007-10-08 16:32:41.000000000 -0400
+++ book-contents.xhtml	2007-10-03 16:07:31.000000000 -0400
@@ -17,6 +17,7 @@
 
   <body>
 
+<a name="chapter0"/>
 <h3>Chapter 0: Introduction to Axiom</h3>
 <a href="section-0.1.xhtml">0.1 Introduction to Axiom</a><br/>
 <a href="section-0.1.xhtml#subsec-0.1.1" class="subseccontents">0.1.1 Symbolic Computation</a><br/>
@@ -71,6 +72,7 @@
 <a href="section-0.7.xhtml#subsubsec-0.7.5.2" class="subsubseccontents">0.7.5.2 The <span class="teletype">while</span> loop</a><br/>
 <a href="section-0.7.xhtml#subsubsec-0.7.5.3" class="subsubseccontents">0.7.5.3 The <span class="teletype">for</span> loop</a><br/>
 
+<a name="chapter1"/>
 <h3>Chapter 1: An Overview of Axiom</h3>
 <a href="section-1.0.xhtml">1.0 An Overview of Axiom</a><br/>
 <a href="section-1.1.xhtml">1.1 Starting Up and Winding Down</a><br/>
@@ -101,6 +103,7 @@
 <a href="section-1.15.xhtml#subsec-1.15.1" class="subseccontents">1.15.1 Undo</a><br/>
 <a href="section-1.16.xhtml">1.16 Graphics</a><br/>
 
+<a name="chapter2"/>
 <h3>Chapter 2: Using Types and Modes</h3>
 <a href="section-2.0.xhtml">2.0 Using Types and Modes</a><br/>
 <a href="section-2.1.xhtml">2.1 The Basic Idea</a><br/>
@@ -124,6 +127,7 @@
 <a href="section-2.11.xhtml">2.11 Exposing Domains and Packages</a><br/>
 <a href="section-2.12.xhtml">2.12 Commands for Snooping</a><br/>
 
+<a name="chapter3"/>
 <h3>Chapter 3: Using HyperDoc</h3>
 <a href="section-3.0.xhtml">3.0 Using HyperDoc</a><br/>
 <a href="section-3.1.xhtml">3.1 Headings</a><br/>
@@ -136,6 +140,7 @@
 <a href="section-3.7.xhtml">3.7 Example Pages</a><br/>
 <a href="section-3.8.xhtml">3.8 X Window Resources for HyperDoc</a><br/>
 
+<a name="chapter4"/>
 <h3>Chapter 4: Input Files and Output Styles</h3>
 <a href="section-4.0.xhtml">4.0 Input Files and Output Styles</a><br/>
 <a href="section-4.1.xhtml">4.1 Input Files</a><br/>
@@ -146,6 +151,7 @@
 <a href="section-4.6.xhtml">4.6 IBM Script Formula Format</a><br/>
 <a href="section-4.7.xhtml">4.7 FORTRAN Format</a><br/>
 
+<a name="chapter5"/>
 <h3>Chapter 5: Overview of Interactive Language</h3>
 <a href="section-5.0.xhtml">5.0 Interactive Language</a><br/>
 <a href="section-5.1.xhtml">5.1 Immediate and Delayed Assignments</a><br/>
@@ -170,6 +176,7 @@
 <a href="section-5.5.xhtml">5.5 Creating Lists and Streams with Iterators</a><br/>
 <a href="section-5.6.xhtml">5.6 An Example: Streams of Primes</a><br/>
 
+<a name="chapter6"/>
 <h3>Chapter 6: User-Defined Functions, Macros and Rules</h3>
 <a href="section-6.0.xhtml">6.0 Functions, Macros and Rules</a><br/>
 <a href="section-6.1.xhtml">6.1 Functions vs. Macros</a><br/>
@@ -199,6 +206,7 @@
 <a href="section-6.20.xhtml">6.20 Example: Testing for Palindromes</a><br/>
 <a href="section-6.21.xhtml">6.21 Rules and Pattern Matching</a><br/>
 
+<a name="chapter7"/>
 <h3>Chapter 7: Graphics</h3>
 <a href="section-7.0.xhtml">7.0 Graphics</a><br/>
 <a href="section-7.1.xhtml">7.1 Two-Dimensional Graphics</a><br/>
@@ -238,6 +246,7 @@
 <a href="section-7.2.xhtml#subsec-7.2.10" class="subseccontents">7.2.10 Operations for Three-Dimensional Graphics</a><br/>
 <a href="section-7.2.xhtml#subsec-7.2.11" class="subseccontents">7.2.11 Customization using .Xdefaults</a><br/>
 
+<a name="chapter8"/>
 <h3>Chapter 8: Advanced Problem Solving</h3>
 <a href="section-8.0.xhtml">8.0 Advanced Problem Solving</a><br/>
 <a href="section-8.1.xhtml">8.1 Numeric Functions</a><br/>
@@ -282,6 +291,7 @@
 <a href="section-8.13.xhtml">8.13 Computation of Galois Groups</a><br/>
 <a href="section-8.14.xhtml">8.14 Non-Associative Algebras and Modelling Genetic Laws</a><br/>
 
+<a name="chapter9"/>
 <h3>Chapter 9: Some Examples of Domains and Packages</h3>
 <a href="section-9.1.xhtml">9.1 AssociationList</a><br/>
 <a href="section-9.2.xhtml">9.2 BalancedBinaryTree</a><br/>
@@ -418,6 +428,7 @@
 <a href="section-9.90.xhtml">9.90 XPolynomialRing</a><br/>
 <a href="section-9.91.xhtml">9.91 ZeroDimensionalSolvePackage</a><br/>
 
+<a name="chapter10"/>
 <h3>Chapter 10: Interactive Programming</h3>
 <a href="section-10.0.xhtml">10.0 Interactive Programming</a><br/>
 <a href="section-10.1.xhtml">10.1 Drawing Ribbons Interactively</a><br/>
@@ -431,6 +442,7 @@
 <a href="section-10.9.xhtml">10.9 Functions Producing Functions</a><br/>
 <a href="section-10.10.xhtml">10.10 Automatic Newton Iteration Formulas</a><br/>
 
+<a name="chapter11"/>
 <h3>Chapter 11: Packages</h3>
 <a href="section-11.0.xhtml">11.0 Packages</a><br/>
 <a href="section-11.1.xhtml">11.1 Names, Abbreviations, and File Structure</a><br/>
@@ -444,6 +456,7 @@
 <a href="section-11.9.xhtml">11.9 Testing</a><br/>
 <a href="section-11.10.xhtml">11.10 How Packages Work</a><br/>
 
+<a name="chapter12"/>
 <h3>Chapter 12: Categories</h3>
 <a href="section-12.0.xhtml">12.0 Categories</a><br/>
 <a href="section-12.1.xhtml">12.1 Definitions</a><br/>
@@ -459,6 +472,7 @@
 <a href="section-12.11.xhtml">12.11 Conditionals</a><br/>
 <a href="section-12.12.xhtml">12.12 Anonymous Categories</a><br/>
 
+<a name="chapter13"/>
 <h3>Chapter 13: Domains</h3>
 <a href="section-13.0.xhtml">13.0 Domains</a><br/>
 <a href="section-13.1.xhtml">13.1 Domains vs. Packages</a><br/>
@@ -483,6 +497,7 @@
 <a href="section-13.13.xhtml#subsec-13.13.7" class="subseccontents">13.13.7 Putting It All Together</a><br/>
 <a href="section-13.13.xhtml#subsec-13.13.8" class="subseccontents">13.13.8 Example Queries</a><br/>
 
+<a name="chapter14"/>
 <h3>Chapter 14: Browse</h3>
 <a href="section-14.0.xhtml">14.0 Browse</a><br/>
 <a href="section-14.1.xhtml">14.1 The Front Page: Searching the Library</a><br/>
@@ -540,6 +555,7 @@
 <a href="section-14.3.xhtml#subsubsec-14.3.4.1" class="subsubseccontents">14.3.4.1 Exposure</a><br/>
 <a href="section-14.3.xhtml#subsubsec-14.3.4.2" class="subsubseccontents">14.3.4.2 Threshold</a><br/>
 
+<a name="chapter15"/>
 <h3>Chapter 15: What's New in Axiom Version 2.0</h3>
 <a href="section-15.0.xhtml">15.0 Axiom Packages</a><br/>
 <a href="section-15.1.xhtml">15.1 Important Things to Read First</a><br/>
@@ -572,6 +588,7 @@
 <a href="section-15.6.xhtml">15.6 HyperTex</a><br/>
 <a href="section-15.7.xhtml">15.7 Documentation</a><br/>
 
+<a name="chapter16"/>
 <h3>Chapter 16: Axiom System Commands</h3>
 <a href="section-16.0.xhtml">16.0 Axiom System Commands</a><br/>
 <a href="section-16.1.xhtml">16.1 Introduction</a><br/>
@@ -603,15 +620,19 @@
 <a href="section-16.27.xhtml">16.27 )undo</a><br/>
 <a href="section-16.28.xhtml">16.28 )what</a><br/>
 
+<a name="chapter17"/>
 <h3>Chapter 17: Categories</h3>
 <a href="section-17.1.xhtml">17.1 Axiom Categories</a><br/>
 
+<a name="chapter18"/>
 <h3>Chapter 18: Domains</h3>
 <a href="section-18.1.xhtml">18.1 Axiom Domains</a><br/>
 
+<a name="chapter19"/>
 <h3>Chapter 19: Packages</h3>
 <a href="section-19.1.xhtml">19.1 Axiom Packages</a><br/>
 
+<a name="chapter21"/>
 <h3>Chapter 21: Programs for AXIOM Images</h3>
 <a href="section-21.0.xhtml">21.0 Programs for AXIOM Images</a><br/>
 <a href="section-21.1.xhtml">21.1 images1.input</a><br/>
